codec/unknownproto/unknown_fields.go

```diff
@@ -68,10 +69,9 @@ func RejectUnknownFields(bz []byte, msg proto.Message, allowUnknownNonCriticals
 					Type:         reflect.ValueOf(msg).Type().String(),
 					TagNum:       tagNum,
 					GotWireType:  wireType,
-					WantWireType: protowire.Type(fieldDescProto.WireType()),
+					WantWireType: toProtowireType(fieldDescProto.GetType()),
 				}
 			}
-
```

Comment:
One of the nits I was talking about that help with readability: spaces between cases :-)

Reply:
addressed!